libraries/cerner_tomcat.rb

```diff
@@ -175,20 +175,26 @@ def action_enable
               recursive true
             end
 
-            remote_file "#{Chef::Config[:file_cache_path]}/#{tomcat_file}" do
+            directory '/var/cerner_tomcat' do
```

Comment:
Can we make the path an attribute?

Reply:
Are you ok with defaulting it to /var/cerner_tomcat?

Reply:
Yes thats fine